### PR TITLE
Historical events export fixes

### PR DESCRIPTION
**Pull request by yakkomajuri**
## Changes

- Pass elements with events from a historical export
- Pass UUID (if available) or add Postgres ID to properties
- Add useful properties to historical export events (flag to show that it is from an export, export timestamp, db it came from, etc)
- Minor refactoring

## How did you test this code?

Fixed the existing tests (which were broken), and added a new test for elements.

### CODE REVIEW

File: plugin-server/src/worker/vm/upgrades/historical-export/export-historical-events.ts [hunk 1]
```diff
@@ -129,14 +129,15 @@ export function addHistoricalEventsExportCapability(
                 EVENTS_TIME_INTERVAL,
                 EVENTS_PER_RUN
             )
+            console.log(events)
```
Comment (yakkomajuri) [inline plugin-server/src/worker/vm/upgrades/historical-export/export-historical-events.ts:132]:
```suggestion
            
```

File: plugin-server/.vscode/launch.json [hunk 2]
```diff
@@ -32,6 +35,17 @@
...
+        {
+            "type": "node",
+            "name": "vscode-jest-tests",
+            "request": "launch",
+            "console": "integratedTerminal",
+            "internalConsoleOptions": "neverOpen",
+            "disableOptimisticBPs": true,
+            "program": "${workspaceFolder}/node_modules/.bin/jest",
+            "cwd": "${workspaceFolder}",
+            "args": ["--runInBand", "--forceExit", "--watchAll=false"]
```
Comment (yakkomajuri) [inline plugin-server/.vscode/launch.json:48]: this was accidental, but I take it won't hurt

File: plugin-server/src/worker/vm/upgrades/historical-export/export-historical-events.ts [hunk 3]
```diff
@@ -231,6 +233,7 @@ export function addHistoricalEventsExportCapability(
                 )
             }
             await meta.storage.set(MAX_UNIX_TIMESTAMP_KEY, meta.global.timestampBoundariesForTeam.max.getTime())
+            console.log('###### did dateTo')
```
Comment (guidoiaquinti) [inline plugin-server/src/worker/vm/upgrades/historical-export/export-historical-events.ts:236]: Did we leave this from debugging?

File: plugin-server/src/worker/vm/upgrades/historical-export/export-historical-events.ts [hunk 4]
```diff
@@ -47,13 +47,14 @@ export function addHistoricalEventsExportCapability(
         // the historical export to duplicate them
         meta.global.timestampBoundariesForTeam = timestampBoundaries
 
+        console.log('##########', timestampBoundaries)
```
Comment (tiina303) [inline plugin-server/src/worker/vm/upgrades/historical-export/export-historical-events.ts:50]: keeping these (below too) was deliberate?

**Review by tiina303 — APPROVED** [review]
lgtm

File: plugin-server/src/worker/vm/upgrades/historical-export/export-historical-events.ts (hunk 3 above)
Comment (yakkomajuri) [reply plugin-server/src/worker/vm/upgrades/historical-export/export-historical-events.ts:236]: yup

File: plugin-server/src/worker/vm/upgrades/historical-export/export-historical-events.ts (hunk 4 above)
Comment (yakkomajuri) [reply plugin-server/src/worker/vm/upgrades/historical-export/export-historical-events.ts:50]: not at all, thanks for flagging

**Pull request merged**